modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -48,262 +54,859 @@ curl -X POST -u <administrator>:<password>
+
+----
+curl -v -X POST http://10.143.201.101:8091/pools/default/buckets/testBucket \
+-u Administrator:password \
+-d autoCompactionDefined=false
+----
+
+This disables auto-compaction for the bucket, and removes all auto-compaction-related settings.
+If the call is successful, a `200 OK` notification is returned, with no object.
+
+To _enable_ auto-compaction after bucket creation, the `parallelDBAndViewCompaction` parameter must also be specified; as in the following example, which sets `parallelDBAndViewCompaction` to `false`:
+
+----
+curl -v -X POST http://10.143.201.101:8091/pools/default/buckets/testBucket \
+-u Administrator:password \
+-d autoCompactionDefined=true \
+-d parallelDBAndViewCompaction=false
+----
+
+If successful, the call returns a `200 OK` notification.
+No object is returned.
+
+[#paralleldbandviewcompaction]
+=== parallelDBAndViewCompaction
+
 Specifies whether compaction should occur to documents and view indexes in parallel.
 This is a _global_ setting, which therefore affects _all_ buckets on the cluster.
-The value can either be `true` or `false` (which is the default).
+The value can either be `true` or `false`: one value or the other _must_ be specified.
 If the value is incorrectly specified, the following error-notification is returned: `{"parallelDBAndViewCompaction":"parallelDBAndViewCompaction is invalid"}`.
-+
+
 This parameter-value is ignored if `autoCompactionDefined` is `false` (which is its default value).
 
-* `databaseFragmentationThreshold[percentage]`.
+For examples, see xref:rest-api:rest-bucket-create.adoc#autocompactiondefined[autoCompactionDefined], above.
+
+[#databasefragmentationthresholdpercentage]
+=== databaseFragmentationThreshold[percentage]
+
 Specifies, as a percentage, the level of database fragmentation that must be reached for data compaction to be automatically triggered.
-The assigned value must be an integer.
+The assigned value must be an integer from `0` to `100`.
 The default value is `"undefined"`.
-+
+
 If a value for `databaseFragmentationThreshold[size]` is also specified, data compaction is automatically triggered as soon as the threshold specified by one parameter or the other is reached.
-+
+
 If this parameter is incorrectly specified, an error-notification such as the following is returned: `"databaseFragmentationThreshold[percentage]":"database fragmentation must be an integer"`.
-+
+
 This parameter is ignored if `autoCompactionDefined` is `false` (which is its default value).
 
-* `databaseFragmentationThreshold[size]`.
+[#example-databasefragmentationthresholdpercentage-create]
+==== Example: Specifying a Data Fragmentation Threshold as a Percentage, when Creating
+
+The following example establishes a value for `databaseFragmentationThreshol[percentage]`, and for all other auto-compaction-related parameters, in its creation of a new bucket, named `testBucket`:
```

Comment:
databaseFragmentationThreshol is it missed a 'd'?

Reply:
Thanks, have fixed.